build.sbt

```diff
@@ -102,6 +104,13 @@ lazy val validation = (project in file("validation"))
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
   .settings(
+    assemblyMergeStrategy in assembly := {
+      case "application.conf" => MergeStrategy.concat
+      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
+      case x =>
+        val oldStrategy = (assemblyMergeStrategy in assembly).value
```

Comment:
This settings resolve conflicts for the `.jar` file that is produced by `sbt-assembly`. For instance, our projects all have an `application.conf` configuration file, and the final jar will also have one. In that case you want to concatenate all the settings from the projects that make up the final `jar` file. In the case that you are asking, I put it like that because `panel` should be its own jar eventually, it's an executable that we can use completely separate from the rest of the application. We haven't made that change yet because the need hasn't been there, and because it's not finished yet (i.e. see this [ticket](https://github.com/cfpb/hmda-platform/issues/915))